src/main/kotlin/io/snyk/plugin/ui/SnykBalloonNotifications.kt

```diff
@@ -28,6 +28,7 @@ import io.snyk.plugin.services.SnykTaskQueueService
 import io.snyk.plugin.settings.SnykProjectSettingsConfigurable
 import io.snyk.plugin.snykToolWindow
 import io.snyk.plugin.startSastEnablementCheckLoop
+import org.jetbrains.annotations.Nullable
```

Comment:
@michelkaporin, can you remove the unused import? (detekt-warning)

Reply:
@pavel-snyk detekt does its job, removed :)